code/src/cljs/sixsq/nuvla/ui/edge_detail/views.cljs

```diff
@@ -79,81 +79,142 @@
+
+
 (defn Peripheral
-  [{p-name        :name
-    p-product     :product
-    p-created     :created
-    p-updated     :updated
-    p-descr       :description
-    p-interface   :interface
-    p-device-path :device-path
-    p-available   :available
-    p-vendor      :vendor
-    p-classes     :classes
-    p-indentifier :identifier}]
-  (let [locale (subscribe [::i18n-subs/locale])]
-    [uix/Accordion
-     [ui/Table {:basic "very"}
-      [ui/TableBody
-       (when p-product
-         [ui/TableRow
-          [ui/TableCell "Name"]
-          [ui/TableCell (str p-name " " p-product)]])
-       (when p-descr
-         [ui/TableRow
-          [ui/TableCell "Description"]
-          [ui/TableCell p-descr]])
-       [ui/TableRow
-        [ui/TableCell "Classes"]
-        [ui/TableCell (str/join ", " p-classes)]]
-       [ui/TableRow
-        [ui/TableCell "Available"]
-        [ui/TableCell
-         [ui/Icon {:name "circle", :color (if p-available "green" "red")}]
-         (if p-available "Yes" "No")]]
-       (when p-interface
-         [ui/TableRow
-          [ui/TableCell "Interface"]
-          [ui/TableCell p-interface]])
-       (when p-device-path
-         [ui/TableRow
-          [ui/TableCell "Device path"]
-          [ui/TableCell p-device-path]])
-       [ui/TableRow
-        [ui/TableCell "Identifier"]
-        [ui/TableCell p-indentifier]]
-       [ui/TableRow
-        [ui/TableCell "Vendor"]
-        [ui/TableCell p-vendor]]
-       [ui/TableRow
-        [ui/TableCell "Created"]
-        [ui/TableCell (time/ago (time/parse-iso8601 p-created) @locale)]]
-       [ui/TableRow
-        [ui/TableCell "Updated"]
-        [ui/TableCell (time/ago (time/parse-iso8601 p-updated) @locale)]]]
-      ]
-     :label (or p-name p-product)
-     :title-size :h4
-     :default-open false
-     :icon (case p-interface
-             "USB" "usb"
-             nil)]))
+  [id]
+  (let [locale       (subscribe [::i18n-subs/locale])
+        last-updated (r/atom "1970-01-01T00:00:00Z")
+        button-load? (r/atom false)
+        peripheral   (subscribe [::subs/nuvlabox-peripheral id])]
+    (fn [id]
+      (let [{p-id          :id
+             p-ops         :operations
+             p-name        :name
+             p-product     :product
+             p-created     :created
+             p-updated     :updated
+             p-descr       :description
+             p-interface   :interface
+             p-device-path :device-path
+             p-available   :available
+             p-vendor      :vendor
+             p-classes     :classes
+             p-identifier  :identifier
+             p-serial-num  :serial-number
+             p-video-dev   :video-device
+             p-data-gw-url :local-data-gateway-endpoint
+             p-data-sample :raw-data-sample} @peripheral
+            actions (get-available-actions p-ops)]
+
+        (when (> p-updated @last-updated)
```

Comment:
Here you are comparing 2 string with > function. I don't thing that's what you want.
You have to use the times utils for that

Reply:
ISO 8601 date formats can be compared as strings